lib/assembly-image/image.rb

```diff
@@ -72,5 +62,10 @@ def vips_image
     def srgb?
       vips_image.interpretation == :srgb
     end
+
+    # Does the image include an ICC profile?
+    def has_profile?
+      vips_image.get_fields.include?('icc-profile-data')
+    end
   end
```

Comment:
I think this should live in the jp2_create method - it's not used anywhere else.